File: selenium/UnitTestProject2/UnitTestProject2/Pages/HomePage.cs
using OpenQA.Selenium;

namespace UnitTestProject2.Pages
{
    class HomePage
    {
        IWebDriver driver;

        public HomePage(IWebDriver driver)
        {
            this.driver = driver;
        }

        public void ClickOnLoginLink()
        {

            IWebElement loginLink = driver.FindElement(By.LinkText("Log In"));
            loginLink.Click();
            System.Threading.Thread.Sleep(2*1000);

        }


    }
}

File: newtoursAutomation_Feb2019/src/test/java/com/demoaut/newtours/pages/HomePage.java
package com.demoaut.newtours.pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class HomePage {
	
WebDriver driver;
	
	public HomePage(WebDriver driver) {
		
		this.driver = driver;
		
	}
	
	public void ClickOnRegLink() throws InterruptedException {
		
		WebElement ClickOnRegLink = driver.findElement(By.linkText("REGISTER"));
		ClickOnRegLink.click();
		
	}

}
